ui/src/ensembler/components/table/ListRouterVersionsForEnsemblerTable.js

```diff
@@ -115,44 +115,54 @@ export const ListRouterVersionsForEnsemblerTable = ({
     </EuiCallOut>
   ) : (
     <Fragment>
-      {ensemblerUsedByCurrentRouterVersion ? (
-        currentRouterVersion.data.length > 0 && (
+      {canDeleteEnsembler && results.totalInactiveCount > 0 && (
```

Comment:
The conditional logic here has been simplified - we first check if the ensembler can be deleted - if it can, we'll display the deletion warnings. Otherwise, we'll display the current router versions table, followed by the active router versions table.